### DISCUSSION TITLE
Can we specify a project name for vercel link cli?

### DISCUSSION THREAD

**Discussion by juansalvatore** [vercel/vercel#8930] [Help] [answered]
I'm trying to do something like:
`vercel link --yes --scope my-team-name`
But when using this command it's taking the project name from the parent folder, but instead I want to specify a custom project name that's holding my env variables that I want to fetch.
Is there any way to do something like --project ? or prioritize the vercel.json config with a name? or it would be more intuitive to be able to do: `vercel link --yes --scope my-team-name/(my-project-name | my-project-id)`

For more context, we want to have a script that runs vercel link without prompts and fetches the env variables for multiple projects in a monorepo, and using just `vercel link` with the prompts is way too slow to go through many apps.

Thanks!

**Comment by amyegan** [comment, marked as answer]
I don't know of a way to do that, but I'm going to ask the CLI engineers because they know a lot more about it than I do! I'll share an update when I learn more.

**Reply by juansalvatore** [reply]
Thank you! will wait for the response.

**Reply by amyegan** [reply]
Thanks for waiting! The CLI team let me know that there is a `--project` option for `vercel link`. So you can add `--project=my-project-name` or `-p my-project-name` to the command, along with the `--scope` and `--yes` options you already use, to skip all prompts. They said this information will be added to the [CLI documentation](https://vercel.com/docs/cli/link#) soon, but you can also find it by running `vercel link --help`.

**Reply by juansalvatore** [reply]
awesome! thank you for the quick response
